### DISCUSSION TITLE
DSA

### DISCUSSION THREAD

**Discussion by ankit-akash** [ankit-akash/DSA#12] [Q&A] [answered]
What do you understand by DSA?

**Comment by ankit-akash** [comment]
First person to answer it properly gets marked answer right.

**Comment by Preetiraj3697** [comment, marked as answer]
"You must have seen how a bank’s cashier puts cash in different sections of the drawer. They do that so their transactions would be effortless and secure. Similarly, it becomes easier to perform operations on the data if you store it in a structured way."
Data structures and algorithms(DSA) are the foundational stones and pillars of Computer Sciences. They are the building blocks in all disciplines of software development. 

The efficiency of a software application depends on what data structures and algorithms have been used to create it and thus they’re as integral as the programming language itself. 

Needless to say, every Computer Science student must learn DSA to be a successful software developer in their career.

As important as they are, most of the students in India, however, either skip learning DSA or aren’t able to understand the subject. We understand that learning data structures and algorithms is not as easy a process as learning a programming language.
